CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @DataDog/apm-js
```

Comment:
GitHub seems to be complaining about this. I guess this team needs write access?

Reply:
Fixed.